app/assets/stylesheets/_variables.scss

```diff
@@ -16,20 +16,54 @@ $small-spacing: $base-spacing / 2;
 $base-z-index: 0;
 
 // Colors
-$blue: #477dca;
-$dark-gray: #333;
-$medium-gray: #999;
-$light-gray: #ddd;
-
-// Font Colors
-$base-background-color: #fff;
-$base-font-color: $dark-gray;
-$action-color: $blue;
-
-// Border
-$base-border-color: $light-gray;
+$dark-color: #333;
+$light-color: #fff;
+
+// Background
+$base-background: #fff;
+$base-background-inset: darken($base-background, 4%);
+$base-background-inset-shadow: inset 0 0 2px darken($base-background, 20%);
+$base-font-color-on-background: $dark-color;
+
+// Outline on background
+$base-outline-color: darken($base-background, 15%);
+$base-outline: 1px solid $base-outline-color;
+
+// Box
+$base-box-background: darken($base-background, 3%);
+$base-border-color: darken($base-box-background, 15%);
 $base-border: 1px solid $base-border-color;
+$base-box-background-inset: darken($base-box-background, 5%);
+$base-font-color-on-box: #000;
+$base-box-shadow: 0 1px 2px transparentize($base-font-color-on-box, 0.8);
+$base-box-shadow-inset: inset 0 2px 2px darken($base-box-background, 10%);
+
+// Notification Colors
+$alert-color: #fff6bf;
+$error-color: #fbe3e4;
+$notice-color: #e5edf8;
+$success-color: #e6efc2;
+
+// Action Colors
+$action-color-on-background: #477dca;
```

Comment:
Since a "box" or a "container" can have a completely different background color then the body background, texts or action-colors might not show if we don't allow for the possibility to change them individually. E.g. a white background with a black box. Text on background would need to be dark and text on box would need to be light. Same for the action color: it might need two, or more, different lightness or such.